tests/generate.py

```diff
@@ -64,7 +65,7 @@ def generate(self, yaml, expect_fail=False, extra_args=[], confs=None):
                 with open(os.path.join(self.confdir, f + '.yaml'), 'w') as f:
                     f.write(contents)
 
-        argv = [exe_generate, '--root-dir', self.workdir.name] + extra_args
+        argv = ['catchsegv', exe_generate, '--root-dir', self.workdir.name] + extra_args
```

Comment:
What's this change for? It looks from the man page to be a good idea, but it's not documented in the commit message.

Reply:
(Likewise for some of your other changes in this file.)

Reply:
I did not mean to leave the catchsegv in there -- I inserted it to see why generate was faulting.